logos/src/logos/capacity/calibration_orchestrator.py

```
@@ -0,0 +1,353 @@
+            active: int = getattr(capacity, "active_requests", 0) or 0
+            return active > 0
+        except Exception:
+            # If we can't determine, assume busy to avoid interrupting work.
+            return True
+
+    def _find_uncalibrated_model(self, provider_id: int) -> str | None:
+        """Return the first model on this provider that has no calibration data."""
+        capabilities = self._facade.get_worker_capabilities(provider_id)
+        try:
+            profiles = self._facade.get_model_profiles(provider_id)
+        except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)